Analyzer/test/MyAnalysis.C

```diff
@@ -82,6 +83,8 @@ template<typename Analyze> void run(const std::set<AnaSamples::FileSummary>& vvf
         const bool isSignal = (file.tag.find("_stop") != std::string::npos || file.tag.find("_mStop") != std::string::npos) ? true : false;
         const std::string DeepESMCfg = (runYear == "2016") ? "DeepEventShape_2016.cfg" : "DeepEventShape_2017.cfg";
         const std::string ModelFile = (runYear == "2016") ? "keras_frozen_2016.pb" : "keras_frozen_2017.pb";
+        const bool doQCDCR = false;//bool to determine to use qcd control region
```

Comment:
I think this bool should go into Analyzer/Analyzer/include/Config.h where you make a if statement specifically for analyzers running over the QCD control region. Then you can set it to be true in the if statement and never have to think if I turn this flag on or off.